NetKAN/KSP-Slovakia-flags

```diff
@@ -0,0 +1,21 @@
+{
+    "spec_version" : "v1.2",
+    "identifier"   : "KSP Slovakia flags",
+    "$kref"        : "#/ckan/github/miki-g/KSP-Slovakia-Flags",
+    "$vref"        : "#/ckan/ksp-avc",
```

Comment:
"Can you replace this line with the following?" - This means which line?

in coment is 4 lines, from 2 to 5

(sry, github and English do not know much)